src/gui/README.md

```diff
@@ -93,6 +94,64 @@ Result:
 ```
 
 ## General system checks
+### Network check
+
+```
+URI: /network/connections/default
+
+Method: GET
+```
+
+Example:
+
+```sh
+curl http://127.0.0.1:6420/network/connections/default
+```
+
+Response:
+
+```json
```

Comment:
JSON should be formatted with indentation

Reply:
done

Reply:
Needs to be the same format as the other examples here